pulpcore/cli/common/generic.py

```diff
@@ -414,6 +414,7 @@ def resource_option(*args: Any, **kwargs: Any) -> Callable[[FC], FC]:
     context_table: Dict[str, Type[PulpEntityContext]] = kwargs.pop("context_table")
     capabilities: Optional[List[str]] = kwargs.pop("capabilities", None)
     href_pattern: Optional[str] = kwargs.pop("href_pattern", None)
+    is_command_entity: Optional[bool] = kwargs.pop("is_command_entity", False)
```

Comment:
I don't think, a new argument is needed here. Can we just take `expose_value` for it?
If `expose_value=True` we return a new context, else we find and polulate an existing context.